### ISSUE TITLE
Define focus of this bot as forms, not surveying.

### ISSUE THREAD

**Issue by connorhsm** [voteflux/flux-projects#33]
This issue is mostly to quote my thoughts about highlighting this bot is about filling data about an object where that data may later change, not asking a question like you would on a survey.

I'll also use this issue to track some updates related to this clarification of the focus.

> 
> 
> Struggling to progress with this.
> How can I retrieve a past response to a question (eg, the title of a project) and know what we asked them so that we can then ask them this again to update the response?
> 
> Light bulb moment.
> I'm going about this the wrong way. It shouldn't be: ask a question and receive a response, everything should be a field that the user fills in. This isn't about surveying (asking questions) this is about collecting data (filling a form).
> 
> I recall almost coming upon this back when I was creating the questioning backend. Look at the questions, there is no substance to them.
> 
>     * What is your project title?
> 
>     * What is the start date of your project?
> 
>     * What is the end date of your project?
> 
>     * What is the desciption of your project?
> 
>     * What are the outcomes of your project?
> 
>     * What are the deliverables of your project?
> 
>     * What is the objective of your project?
> 
> 
> They all follow a very similar template, requesting something, what that something is and a few adjustments for phrasing/context. Let's strip all these unnecessary words out and only say it once. "Please fill in the following fields about your new project: Title, Description etc". These become the labels for each data point and respectively, the column names in the database, solving this issue.
> 
> TLDR: We need to focus on filling data, like a form and make sure it's not like a survey where we ask questions.
